dashboard/test/ui/runner.rb

```diff
@@ -663,7 +665,10 @@ def tag(tag, run = true)
   " -t #{tag}"
 end
 
+def skipped_tags = $skipped_tags ||= Set.new
```

Comment:
I learned about Ruby "end-less methods" a few days ago (doesn't need an end statement: https://www.bigbinary.com/blog/ruby-3-adds-endless-method-definition), and this was a cool place to use them. They were experimental (but no flag needed) in 3.0, but are no longer experimental, so they're safe to use.

While I wish runner.rb was not architected to use $globals, it is, and I'm rolling with it here, but I like how I only had to reference the global once using this pattern.

Reply:
Oh, nice; TIL! I do appreciate me a lightweight method declaration syntax